### PR TITLE
Bugfix/fix dataverse search

### PR DESCRIPTION
**Pull request by JonathanThomassin**

### CODE REVIEW

**Review by GabrielFG — APPROVED** [review]
lgtm

File: src/Solution/Solution/Entities/opc_pafile/SavedQueries/{5fc9277a-adb3-4f6f-9940-277785360e5b}.xml [hunk 1]
```diff
@@ -60,18 +59,16 @@
...
             <condition attribute="opc_filetype" operator="like" value="{0}" />
             <condition attribute="opc_filenumber" operator="like" value="{0}" />
-            <condition attribute="opc_dispositioninvestigation" operator="like" value="{0}" />
-            <condition attribute="opc_dispositionintake" operator="like" value="{0}" />
             <condition attribute="opc_dispositionfollowup" operator="like" value="{0}" />
+            <condition attribute="opc_dispositionintake" operator="like" value="{0}" />
+            <condition attribute="opc_dispositioninvestigation" operator="like" value="{0}" />
             <condition attribute="opc_dispositionearlyresolution" operator="like" value="{0}" />
             <condition attribute="opc_descriptionfollowup" operator="like" value="{0}" />
-            <condition attribute="opc_descriptioncommon" operator="like" value="{0}" />
```
Comment (GabrielFG) [inline src/Solution/Solution/Entities/opc_pafile/SavedQueries/{5fc9277a-adb3-4f6f-9940-277785360e5b}.xml:71]: We should bring back the description to enable the search on it again.

Comment (JonathanThomassin) [reply src/Solution/Solution/Entities/opc_pafile/SavedQueries/{5fc9277a-adb3-4f6f-9940-277785360e5b}.xml:71]: done

**Review by GabrielFG — APPROVED** [review]
lgtm2

**Pull request merged**